tests/app/main/views/test_communications.py

```diff
@@ -39,88 +52,198 @@ def test_get_page_should_only_be_accessible_to_specific_user_roles(self, role, e
         assert actual_code == expected_code, "Unexpected response {} for role {}".format(actual_code, role)
 
     def test_page_shows_empty_messages_if_no_files_uploaded(self):
-        self.s3.return_value.list.side_effect = ([], [])  # Empty lists for both communication and clarification files
-        self.data_api_client.get_framework.return_value = {"frameworks": {"status": "open"}}
+        self.s3.return_value.list.side_effect = lambda *args, **kwargs: []  # return a fresh empty list every time
         response = self.client.get("/admin/communications/{}".format(self.framework_slug))
         document = html.fromstring(response.get_data(as_text=True))
-        file_upload_messages = document.xpath('//p[@class="file-upload-existing-value"]')
+        file_upload_messages = document.xpath('//p[@class="summary-item-no-content"]')
 
         assert len(file_upload_messages) == 2
-        assert file_upload_messages[0].text_content().strip() == "No communications have been uploaded yet"
-        assert file_upload_messages[1].text_content().strip() == "No clarification answers have been uploaded yet"
+        assert file_upload_messages[0].text_content().strip() == "No communications files"
+        assert file_upload_messages[1].text_content().strip() == "No clarification answers"
```

Comment:
Just out of curiosity, what was the reasoning behind changing the empty message?

Reply:
It's the whole "yet" thing. _Before_ it was possible to say that because the only way a section could have no entries is in the initial state (unless someone had fiddled with the bucket manually). But now a section can be empty because it's had its files _deleted_, and in that case the message is misleading.